opm/autodiff/MissingFeatures.cpp

```diff
@@ -55,6 +89,14 @@ namespace MissingFeatures {
             "VAPPARS", "VISCREF", "WATVISCT",
             "WPAVE", "WPIMULT", "WPITAB", "WTEMP",
             "WTEST", "WTRACER", "ZIPPY2" };
+        std::multimap<std::string, PartiallySupported<std::string> > string_options;
```

Comment:
@joakim-hove 

If you mean the last type in this line, I guess the answer is no, since the compiler reports error.
 `` addUnsupported<ParserKeywords::COMPORD, ParserKeywords::COMPORD::ORDER_TYPE, std::string>``